docs/115-release.md

```diff
@@ -23,6 +23,10 @@ release:
   # If set to true, will mark the release as not ready for production.
   # Default is false.
   prerelease: true
+
+  # Optional template to name the release
+  # Default is the version number
+  name_template: "{{.ProjectName}}-v{{.Version}}"
```

Comment:
the default name until now was the current tag, right? Can we keep the default as it was before?

Reply:
You’re right. I forgot that I had been tagging my own without a “v” so I was thinking that the “v” was being stripped. I can fix this.